WordPress/Classes/Utility/WPError.swift

```diff
@@ -0,0 +1,37 @@
+
+import Foundation
+import WordPressFlux
+
+extension WPError {
```

Comment:
Following the existing conventions in the code base this file should be called `WPError+Swift.swift`